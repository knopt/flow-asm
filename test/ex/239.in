11
2
6.287832327293149
1.628384173783548 -2.491623939029555 5.98710495794402 8.375685089114125 -7.294196866409575 8.529072846926258 -4.467630550842905 7.9196184223280355 -3.0305453787506575 -3.4730683520531365 2.9990808976730747 
2.481407818087657 -5.086068851169632 -4.0835131458469025 0.7615885506074669 -8.351819138439893 7.957645871447589 8.37171572763856 -7.364150180007199 2.587001051005773 -2.8188695798159724 5.508103000073646 

0.9269132591573133 3.709723757591883 
