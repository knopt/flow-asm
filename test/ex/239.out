15.668436645886521 38.370451518548165 -180.27433900632184 -132.86081806788212 135.23866991311894 -200.70224932829663 236.113172576669 -163.22838836662297 73.89545823686619 41.5276343917903 -58.502695187400406 
-4.933218296212526 101.02974496615052 62.944475165248015 51.02973482542336 160.78165368963846 -186.90152826736679 -73.97410750943646 205.89494023356787 -61.775755497748165 25.72786440983416 -119.09891422646916 
